### DISCUSSION TITLE
I'm Facing This Issue, Seems like it is From Mongodb

### DISCUSSION THREAD

**Discussion by mduvernon** [reactioncommerce/reaction#6729] [Q&A]
Mongo using docker 

Does not work with MongoDB 5  root user (login and pwd)

It seems to work with MongoDB 4.2 with root user (user + pwd)

![image](https://user-images.githubusercontent.com/12007807/208162094-fc30fb14-f03f-424d-b476-79f8b410f7cc.png)

**Comment by brent-hoover** [comment]
Hey, the current released docker fire only works with Mongo < 5 however we just merged changes yesterday to support version 4 of the driver which supports MongoDB 5. Please try with the latest trunk and see if that fixes your issue
